Solutions/DigitalGuardianDLP/Analytic Rules/DigitalGuardianPossibleProtocolAbuse.yaml

```diff
@@ -0,0 +1,29 @@
+id: a374a933-f6c4-4200-8682-70402a9054dd
+name: Digital Guardian - Possible protocol abuse
+description: |
+  'Detects possible protocol abuse.'

```

Comment:
Adding SMTP in the description will be better.

Reply:
added protocol name